ansible/image.yml

```diff
@@ -0,0 +1,19 @@
+---
+- import_playbook: setup.yml
```

Comment:
It seems the scripts setup.yml and monitoring.yml are used both:
* When creating images (this PR - packer branch only);
* At midday every day - on each host - from master branch (see [master](https://github.com/aeternity/infrastructure/blob/6f100077951b7f73384f0a90a6eeb63708c5d145/.circleci/config.yml#L204)).

So these scripts have double use.

I am concerned re how the two branches (master and packer) will be updated when these scripts (setup.yml and monitoring.yml) change. What workflow did you have in mind?

Reply:
The workflow that I have in mind is to run `packer` manually (pushing to remote branch or locally).

The `packer` build images is just a base, that would speedup instances initial provisioning.

Then the daily (or master) run of the playbooks keep it in sync with latest changes.


The workflow is the same as we have now, with a different base instance image which will be setup in advance - "build" time instead of "provisioning" time.

Reply:
So, in conclusion - the packer base image does not need to be "very" up-to-date, of course we'd keep it, but even it's forgotten it's not a problem.

Reply:
So it looks like the problem addressed and the workflow is:

* If DevOps needs to instantiate env with more than e.g. 3 nodes, prefer using recent packer-based images (AWS, openstack).
* In order to refresh the package-based images (AWS, openstack), force-push recent `master` branch to `packer` branch.

Right? If yes, `packer` branch shall *not* be protected (as `env/dev1` in epoch repo).

Could you please README.md ?

Reply:
yep, see b161e50